README.md

```diff
@@ -1 +1,110 @@
+
+The repository contains the implemenation for the GDI starter kit, as well as a demo that can be used for understanding the product. Both solutions are packaged in the `docker-compose-htsget.yml` file. The major difference between the two implementations, is that the demo one can run on it's own, while the starter kit version requires some of the other GDI products in order to be functional.
+
+More details regarding that and the way to run the services can be found in the sections below. After deciding whether you would like to run the demo or starter kit version, follow the instructions on the respective section.
+
+## Disclaimer
+**Important Note:** The current implementation of the htsget server **does not** enable for partial retrieval of a file. Therefore, only full files should be requested. This feature will be developed in future versions of the product.
+
+
+## HTSGET Configuration
+The configuration for the htsget server should rely in the a folder called `config-htsget`. Detailed description of the configuration options can be found in the [reference implementation repository](https://github.com/ga4gh/htsget-refserver#setup---native).
+
+## Running the services - Demo
+The demo profile of the docker compose, apart for the htsget server, contains a minio S3 that can be used as a storage backend and a data downloader that extracts a file in the `output` directory. Specifically, the S3 storage is being populated with the `NA12878.bam` file included in the `demo-data` folder of this repository. The data downloader is then setting the required environments variables and using the `samtools`, it makes a request to the `htsgest server`, which gives access to the requested file.
+
+In order to run the demo version of the htsget, first follow the instructions about removing tls at the bottom of this file and then run
+```sh
+docker compose -f docker-compose-htsget.yml --profile demo up
+```
+Once all the services are finished, you should be able to find the `output` folder in the root of the repository, containing the `NA12878.bam` file.
+
+If you are interested in the commands used for extracting the data, you can check the `data_downloader` section of the `docker-compose-htsget.yml` file.
+
+## Running the services - Starter kit version
+The htsget product of the starter kit depends on the storage-and-interfaces. Specifically, the data served has to be ingested and stored in the archive included in the [storage-and-interfaces repository](https://github.com/GenomicDataInfrastructure/starter-kit-storage-and-interfaces). Therefore, in order to test the implementation, the two docker compose files (the storage-and-interfaces and the htsget) need to be started together, so that they share the same network and the different containers have access between them.
```

Comment:
```suggestion
The htsget product of the starter kit depends on the storage-and-interfaces product. Specifically, the data served has to be ingested and stored in the archive included in the [storage-and-interfaces repository](https://github.com/GenomicDataInfrastructure/starter-kit-storage-and-interfaces). Therefore, in order to test the implementation, the two docker compose files (the storage-and-interfaces and the htsget) need to be started together, so that they share the same network and the different containers have access between them.
```